scripts/diff-features.js

```diff
@@ -68,7 +74,7 @@ function getEnumerationFromGithub(ref) {
     encoding: 'utf-8',
   }).trim();
   const workflowRun = execSync(
-    `gh api /repos/:owner/:repo/actions/workflows/${ENUMERATE_WORKFLOW}/runs?per_page=100 --jq '.workflow_runs[] | select(.head_sha=="${hash}") | .id'`,
+    `gh api /repos/:owner/:repo/actions/workflows/${ENUMERATE_WORKFLOW}/runs?per_page=100 --jq '[.workflow_runs[] | select(.head_sha=="${hash}") | .id] | first'`,
```

Comment:
This wasn't required, but it was a bug that slowed testing of the release notes generation.

Sometimes, a commit can have multiple workflow runs (e.g., on merge and again on being tagged) and the original `jq` query assumed there would be only one, which caused this function to fall back to the slower diffing process.

I've fixed it for my convenience. Happy to move this to a separate PR, if desired.